clvm/operators.py

```diff
@@ -50,7 +50,8 @@ def __new__(class_, d: Dict):
     def __call__(self, op: bytes, arguments: CLVMObject) -> Tuple[int, CLVMObject]:
         f = self.get(op)
         if f is None:
-            f = lambda args: self.unknown_op_handler(op, args)
+            def f(args):
+                return self.unknown_op_handler(op, args)
```

Comment:
apparently default flake8 prefers this. I'm trying hard not to judge :)

Reply:
I guess `lambda` requires understanding of functional programming style, which is a tad meta.